CMakeLists.txt

```diff
@@ -228,6 +228,15 @@ if ( NOT ${WRF_CASE} IN_LIST WRF_CASE_OPTIONS )
   message( FATAL_ERROR "WRF Case option not recognized : ${WRF_CASE}" )
 endif()
 
+# handle Hydro options
+if ( ${WRF_HYDRO} )
+  set(ENABLE_HYDRO TRUE)
+endif()
+if ( ENABLE_HYDRO )
+  set(WRF_HYDRO 1)
+  set(USE_MPI TRUE)
```

Comment:
This should use `CACHE BOOL "<some description - I typically used 'Required by configuration" FORCE` to ensure any cached setting of this will be changed.

Additionally, `USE_*` and `ENABLE_*` typically have option `ON`/`OFF` - this is to make them consistent and identify them as option control

Reply:
Made these changes, good to know!